src/components/Navigation/Navigation.scss

```diff
@@ -99,7 +99,11 @@ $disabled-fade: 0.6;
 }
 
 .SecondaryAction {
-  @include recolor-icon(color('ink', 'lightest'), color('white'));
+  @include recolor-icon(
+    color('ink', 'lightest'),
+    color('white'),
+    filter('ink', 'lightest')
```

Comment:
Why did you add these filters?

Reply:
Looks like this uses the revised `recolor-icon` mixin to handle all cases here: the first 2 params handle the case where the icon is an inline SVG, the other when the icon is an `img`.